languages/bash/reference/README.md

```diff
@@ -0,0 +1,60 @@
+# Bash Reference
+
+## Types
+
+- [string][type-string]
+- [float][type-float]
+- [integer][type-int]
+- [double][type-double]
+- [boolean][type-boolean]
+- [regular expression][type-regex]
+- [function][type-function]
+- [array][type-array]
+- [associative array][type-map]
+
+[type-array]: ../../../reference/types/array.md
+[type-boolean]: ../../../reference/types/boolean.md
+[type-double]: ../../../reference/types/double.md
+[type-float]: ../../../reference/types/floating_point_number.md
+[type-function]: ../../../reference/types/function.md
+[type-int]: ../../../reference/types/integer.md
+[type-map]: ../../../reference/types/map.md
+[type-regex]: ../../../reference/types/regular_expression.md
+[type-string]: ../../../reference/types/string.md
+
+
+## Concepts
+
+- [Functions][concept-functions]
+- [Rest parameters][concept-rest-parameters]
+- [Variables][concept-variables]
+- [Comments][concept-comments]
+- [Assignment][concept-assignment]
+- [Arithmetic][concept-arithmetic]
+- [ASCII][concept-ascii]
+- [Pattern matching][concept-pattern-matching]
+- [Conditionals][concept-conditionals]
+- [Boolean logic][concept-boolean-logic]
+- [Loops][concept-loops]
+- [Functions][concept-functions]
+- [Evaluation][concept-evaluation]
+- [Scope][concept-scope]
+- Package Management
+- [Sameness][concept-sameness]
+
+[concept-arithmetic]: ../../../reference/concepts/arithmetic.md
+[concept-ascii]: ../../../reference/concepts/ascii.md
+[concept-assignment]: ../../../reference/concepts/assignment.md
+[concept-boolean-logic]: ../../../reference/concepts/boolean_logic.md
+[concept-comments]: ../../../reference/concepts/comments.md
+[concept-conditionals]: ../../../reference/concepts/conditionals.md
+[concept-evaluation]: ../../../reference/concepts/evaluation.md
+[concept-expressions]: ../../../reference/concepts/expressions.md
+[concept-functions]: ../../../reference/concepts/functions.md
+[concept-loops]: ../../../reference/concepts/loops.md
+[concept-pattern-matching]: ../../../reference/concepts/pattern_matching.md
+[concept-rest-parameters]: ../../../reference/concepts/rest_parameters.md
+[concept-scope]: ../../../reference/concepts/scope.md
+[concept-variables]: ../../../reference/concepts/variables.md
+[concept-sameness]: ../../../reference/concepts/sameness.md
+
```

Comment:
This section seems to be "OK" as it sits, though not verified by clicking through to the content.

Would it be worth alphabetizing these so that the concepts can be found more easily on the page?